src/backend.rs

```diff
@@ -133,6 +139,8 @@ pub struct BackendHandler<T, P> {
     /// The block to fetch data from.
     // This is an `Option` so that we can have less code churn in the functions below
     block_id: Option<BlockId>,
+    /// Marker identifying whether the RPC provider supports `eth_getAccount`
+    get_account_mode: OnceLock<GetAccountMode>,
```

Comment:
why does this have to be a oncelock?

Reply:
We only want to set it once on the first request, and we also don't need to make `get_account_req(&mut self, address)`

Reply:
Ahh wait, get_account_req is being used in request_account which takes &mut self so I guess it's fine to make it get_account_req(&mut self, address)

Reply:
We're also setting the value inside Box::pin async fut